app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,221 @@
+/* Global Styles */
+
+html,
+body {
+  height: 100%;
+}
+
+body {
+  padding-top: 42px; /* Required padding for .navbar-fixed-top. Remove if using .navbar-static-top. Change if height of navigation changes. */
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-weight: 400;
+}
+
+.navbar {
+  font-weight: 400;
+}
+
+.page-header {
+  border-bottom: none;
+  color: white;
+  float: right;
+  font-weight: 500;
+}
+
+.featured {
+  color: black;
+  float: none;
+}
+
+.main-image {
+  background: url("dew-drops-carnivorous-plant.jpg") no-repeat center center;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+  height: 362px;
+}
+
+#index-foods,
+#index-accessories,
+#index-plants {
+  background: no-repeat center center;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+  height: 200px;
+
+  .btn {
+    margin-top: 130px;
+  }
+}
+
+#index-foods {
+  background: url("frog-in-pitcher-plant.jpg") no-repeat center center;
+  background-size: 100%;
+}
+
+#index-plants {
+  background: url("venus_flytrap.jpg") no-repeat center center;
+  background-size: 100%;
+}
+
+#index-accessories {
+  background: url("planters.jpg") no-repeat center center;
+  background-size: 100%;
+}
+
+#index-foods,
+#index-accessories,
+#index-plants {
+  height: 200px;
+}
+
+.img-portfolio {
+  margin-bottom: 30px;
+}
+
+.img-hover:hover {
+  opacity: 0.8;
+}
+
+/* Home Page Carousel */
+
+.carousel {
+  height: 50%;
+
+  .fill {
+    background-position: center;
+    background-size: cover;
+    height: 100%;
+    width: 100%;
+  }
+}
+
+.carousel .item,
+.carousel .item.active,
+.carousel .carousel-inner {
+  height: 100%;
+}
+
+#ProductCarousel .carousel,
```

Comment:
Avoid using id selectors<br>Selector `ProductCarousel` should be written in lowercase with hyphens<br>Selector `ProductCarouselSm` should be written in lowercase with hyphens